src/main/java/cloud/fogbow/common/models/CloudStackUser.java

```diff
@@ -1,7 +1,20 @@
 package cloud.fogbow.common.models;
 
+import java.util.HashMap;
+
 public class CloudStackUser extends CloudUser {
-    public CloudStackUser(String userId, String userName, String tokenValue) {
+    private HashMap<String, String> cookieHeader;
```

Comment:
Maybe 'cookieHeaders' is a better name for this field since it is a collection.

Reply:
will do